fictadvisor-api/src/v2/api/controllers/ScheduleController.ts

```diff
@@ -267,15 +258,23 @@ export class ScheduleController {
     NoPermissionException:
       You do not have permission to perform this action`,
   })
+  @ApiParam({
+    name: 'groupId',
+    type: String,
+    required: true,
+  })
+  @ApiEndpoint({
+    summary: 'Create an event',
+    guards: TelegramGuard,
+  })
```

Comment:
this endpoint must be accessible from bot and also from web, so return permission